### PR TITLE
doc(volume): max volume size for the file system

### PR DESCRIPTION
**Pull request by mantissahz**
Descriptions of the maximum volume size for file systems.

Ref: longhorn/longhorn#7423

### CODE REVIEW

File: content/docs/1.6.0/volumes-and-nodes/expansion.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ Longhorn will try to expand the file system only if:
 - The filesystem used in the Longhorn volume is one of the following:
     - ext4
     - xfs
+- The expanded size should be smaller than the maximum file size of the file system, e.g., 16TiB for the `ext4` file system.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/volumes-and-nodes/expansion.md:80]:
```suggestion
The expanded size must be less than the maximum file size allowed by the file system (for example, 16TiB for `ext4`).
```

File: content/docs/1.6.0/volumes-and-nodes/volume-size.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ In this section, you'll have a better understanding of concepts related to volum
 - The reason we call this field as "nominal size" is that Longhorn replicas are using [sparse files](https://wiki.archlinux.org/index.php/Sparse_file) to store data and this value is the apparent size of the sparse files (the maximum size to which they may expand). The actual size used by each replica is not equal to this nominal size.
 - Based on this nominal size, the replicas will be scheduled to those nodes that have enough allocatable space during the volume creation. (See [this doc](../node-space-usage) for more info about node allocation size.)
 - The value of nominal size determines the max available space when the volume is in use. In other words, the current active data size hold by a volume cannot be greater than its nominal size.
+- The maximum volume size depends on the disk's file system, e.g., 16383 GiB for the `ext4` file system.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/volumes-and-nodes/volume-size.md:15]:
```suggestion
The maximum volume size is based on the disk's file system (for example, 16383 GiB for `ext4`).
```

**Review by jillian-maroket — COMMENTED** [review]
Let me know if you have concerns about the changes.

File: content/docs/1.6.0/volumes-and-nodes/expansion.md (hunk 1 above)
Comment (mantissahz) [reply content/docs/1.6.0/volumes-and-nodes/expansion.md]: Modified. Thanks.

File: content/docs/1.6.0/volumes-and-nodes/volume-size.md (hunk 2 above)
Comment (mantissahz) [reply content/docs/1.6.0/volumes-and-nodes/volume-size.md]: Modified. Thanks.

**Review by jillian-maroket — APPROVED** [review]

**Review by innobead — CHANGES_REQUESTED** [review]
Please update 1.5.4 and 1.4.5 doc as well, as the fix should be backported.

**Comment by mantissahz** [comment]
>Please update 1.5.4 and 1.4.5 doc as well

Updated 1.5.4 and 1.4.5 documents. Thanks.

**Review by innobead — APPROVED** [review]

**Comment by ejweber** [comment]
I would suggest also updating any unarchived docs with this information. Users may be on an older version (e.g. 1.4.3) and only searching that relevant doc to understand why new volume attachment and/or expansion is failing. Even though the older versions won't have any enforcement, it helps troubleshooting.

**Pull request merged**